frontend/src/styles.css

```diff
@@ -185,6 +191,15 @@ fieldset > :not(.field-separator) {
   margin: auto;
 }
 
+/*
+Own collab manager card styles
+*/
+@layer components {
```

Comment:
Is there a reason for using the `layer` CSS feature? We don't use it anywhere else, as far as I know. 
Given that there is only one layer, it doesn't seem to have any effect.